tools/pubsys/src/main.rs

```diff
@@ -39,8 +40,17 @@ fn run() -> Result<()> {
     // Parse and store the args passed to the program
     let args = Args::from_args();
 
-    // SimpleLogger will send errors to stderr and anything less to stdout.
-    SimpleLogger::init(args.log_level, LogConfig::default()).context(error::LoggerSnafu)?;
+    match args.log_level {
+        // Set log level for AWS SDK to error to reduce verbosity.
+        LevelFilter::Info => Builder::new()
+            .filter_level(args.log_level)
+            .filter(Some("aws_config"), LevelFilter::Error)
+            .filter(Some("aws_smithy"), LevelFilter::Error)
+            .init(),
+
+        // Set the supplied log level across the whole crate.
+        _ => Builder::new().filter_level(args.log_level).init(),
```

Comment:
If I don't explicitly set the logging level for `pubsys`, I think I'd still want to skip out on all the extraneous `info` level log from the `aws-rust-sdk` crates.

Reply:
Since `pubsys` will get a default value of `INFO` for `args.log_level`, I think this works as is. The filtering will apply even if the level isn't explicitly set, and if one were passed we would want that intentional log level applied to everything.